### DISCUSSION TITLE
HLS load test

### DISCUSSION THREAD

**Discussion by maruf-cs** [OvenMediaLabs/OvenMediaEngine#1697] [Help]
I’m looking for some recommendations on a good HLS load testing tool to monitor and improve the performance of my edge. Also, if anyone has tips on how to figure out the right number of workers based on viewer count, I’d love to hear them! Thanks so much for your help!

**Comment by getroot** [comment]
We have done HLS load testing using JMeter in the past.
https://jmeter.apache.org/
